src/components/Accordion.js

```diff
@@ -40,6 +41,10 @@ const Accordion = React.createClass({
       item.onOpened();
```

Comment:
@jasonLaster `item.onOpened()` is only triggered when the item is opened. Should we create a new event such as `item.onHeaderClicked()` which fires every time the item is clicked, or do we go with making `item.onOpened()` fire on click and just pass the current `opened` state to it?